sample-kuryr-config.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: "networkoperator.openshift.io/v1"
+kind: "NetworkConfig"
+metadata:
+  name: "default"
+  namespace: cluster-network-operator
+spec:
+  serviceNetwork: "10.3.0.0/24"
+  clusterNetworks:
+    - cidr: "10.4.0.0/16"
+      hostSubnetLength: 9
```

Comment:
I think we are now using HostPrefix instead of hostSubnetLength

Reply:
Thanks, I forgot to update the sample.